assets/locales/en.json

```
@@ -105,5 +105,10 @@
     "title": "Port is busy",
     "message": "The port { port } is not available. Do you want to use { alt } instead?",
     "action": "Use port { alt } instead"
+  },
+  "multipleBusyPortsDialog": {
+    "title": "Ports are busy",
+    "message": "Some of the API or Gateway addresses cannot be binded to because ports are busy.",
```

`bound` or `bind`, I think `binded` is not a word!

```suggestion
    "message": "Cannot bind to one or more of the API or Gateway addresses because the ports are busy.",
```

---

Ahah, thanks!!